### PR TITLE
fix(service): on project namespace create service error

### PR DESCRIPTION
**Pull request by Muzry**
#### What type of this PR

Add one of the following kinds:
/kind bug

#### What this PR does / why we need it:

- fix(service): on project namespace create service error

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @luobily @sixther-dc 

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/scheduler/executor/plugins/k8s/k8s.go [hunk 1]
```diff
@@ -79,11 +79,14 @@ const (
...
-	ProjectNamespaceServiceNameNameKey = "PROJECT_NAMESPACE_SERVICE_NAME"
-	ProjectNamespace                   = "PROJECT_NAMESPACE"
+	LabelServiceGroupID = "servicegroup-id"
+)
+
+type RuntimeServiceOperator string
+
+const (
+	RuntimeServiceKeep   RuntimeServiceOperator = "Keep"
+	RuntimeServiceDelete RuntimeServiceOperator = ""
```
Comment (luobily) [inline modules/scheduler/executor/plugins/k8s/k8s.go:89]: ambiguous

Comment (Muzry) [reply modules/scheduler/executor/plugins/k8s/k8s.go]: update RuntimeServiceKeep to RuntimeServiceRetain
update RuntimeServceDelete value from "" to "Delete"

**Review by luobily — APPROVED** [review]
/approve

**Pull request merged**